.nvmrc

```diff
@@ -0,0 +1 @@
+14.21.3
```

Comment:
node14 is already EOL, do you want to bump this to node16 instead?

Reply:
Yes, done in 422579b11b8dff5c4fdc65dfdeda1d80ab030edc